packages/client/components/Transactions.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { Text, HStack, Box, VStack } from "native-base";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { StyleSheet } from "react-native";
+import { Transaction } from "@cryptify/common/src/domain/entities/transaction";
+import { faCircleArrowDownLeftCustom } from "./icons/faCircleArrowDownLeftCustom";
+import { faCircleArrowUpRightCustom } from "./icons/faCircleArrowUpRightCustom";
+
+type Props = {
+    transactions: Transaction;
```

Comment:
In english when there is an 's' at the end of a word it mean plural but here we only deal with a single 'transaction' so there wouldn't be an 's'. It is singular.

```suggestion
    transaction: Transaction;
```